Makefile

```diff
@@ -4,7 +4,7 @@
 # Export all variables to sub-makes
 export
 
-GLOOE_VERSION=0.18.6
```

Comment:
what is the significance of this?

Reply:
That is the version of GlooE you want to build the plugins for. It downloads the dependency information for the specified version and compares them to the ones in the local `Gopkg.lock`.

Reply:
Right now you can always just build the plugins for one release of GlooE at a time. Still have to think more about what the versioning use cases are and how to best implement them.